sympy/series/formal.py

```diff
@@ -901,6 +905,16 @@ def compute_fps(f, x, x0=0, dir=1, hyper=True, order=4, rational=True,
     return _compute_fps(f, x, x0, dir, hyper, order, rational, full)
 
 
+class Coeff(Function):
+    """
+    Coeff(p, x, n) represents the nth coefficient of the polynomial p in x
+    """
+    @classmethod
+    def eval(cls, p, x, n):
+        if p.is_polynomial(x) and n.is_integer:
```

Comment:
`n.is_integer` may not necessarily be Boolean. Is it possible to use `fuzzy_and`?

Reply:
What will happen if n is a float? not a polynomial? Can this class be accessed by non-polynomials?

Reply:
This class can't be accessed by non-polynomials. And moreover, FPS is a generalization of a polynomial, so `n` being float is out of question.